cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -1519,335 +1519,372 @@ function Start_Containerd {
+  MustDownload-File -OutFile $tmp_dir\td.zip -URLs $url
 
-  # Install the record-reformer plugin.
-  Start-Process "$STACKDRIVER_ROOT\LoggingAgent\Main\bin\fluent-gem" `
-      -ArgumentList "install","fluent-plugin-record-reformer" `
-      -Wait
+  # Start the installer silently. This automatically starts the
+  # "$LOGGINGAGENT_SERVICE" service.
+  Log-Output 'Invoking Logging agent installer'
+  cd $tmp_dir
+  Expand-Archive $tmp_dir\td.zip
+  mv .\td\td-agent-bit-1.5.1-win64\ $LOGGINGAGENT_ROOT
+  cd $LOGGINGAGENT_ROOT
+
+  sc.exe create $LOGGINGAGENT_SERVICE binpath= "${LOGGINGAGENT_ROOT}\bin\fluent-bit.exe -c \fluent-bit\conf\fluent-bit.conf"
+  sc.exe failure $LOGGINGAGENT_SERVICE reset= 30 actions= restart/5000
+  sc.exe query $LOGGINGAGENT_SERVICE
+
+  # Download fluent bit exporter
+  New-Item $LOGGINGEXPORTER_ROOT -ItemType 'directory' -Force | Out-Null
+  MustDownload-File `
+      -OutFile $LOGGINGEXPORTER_ROOT\flb-exporter.exe `
+      -URLs 'https://storage.googleapis.com/gke-release/winnode/fluentbit-exporter/v0.10.3/flb-exporter-v0.10.3.exe'
 
-  # Install the multi-format-parser plugin.
-  Start-Process "$STACKDRIVER_ROOT\LoggingAgent\Main\bin\fluent-gem" `
-      -ArgumentList "install","fluent-plugin-multi-format-parser" `
-      -Wait
+  sc.exe create  $LOGGINGEXPORTER_SERVICE  binpath= "${LOGGINGEXPORTER_ROOT}\flb-exporter.exe --kubernetes-separator=_ --stackdriver-resource-model=k8s --enable-pod-label-discovery --logtostderr --winsvc  --pod-label-dot-replacement=_"
+  sc.exe query $LOGGINGEXPORTER_SERVICE
 
   Remove-Item -Force -Recurse $tmp_dir
 }
 
-# Writes the logging configuration file for Stackdriver. Restart-LoggingAgent
+# Writes the logging configuration file for Logging agent. Restart-LoggingAgent
 # should then be called to pick up the new configuration.
 function Configure-LoggingAgent {
-  $fluentd_config_dir = "$STACKDRIVER_ROOT\LoggingAgent\config.d"
-  $fluentd_config_file = "$fluentd_config_dir\k8s_containers.conf"
+  $fluentbit_config_file = "$LOGGINGAGENT_ROOT\conf\fluent-bit.conf"
+  $FLUENTBIT_CONFIG | Out-File -FilePath $fluentbit_config_file -Encoding ASCII
+  Log-Output "Wrote logging config to $fluentbit_config_file"
 
-  # Create a configuration file for kubernetes containers.
-  # The config.d directory should have already been created automatically, but
-  # try creating again just in case.
-  New-Item $fluentd_config_dir -ItemType 'directory' -Force | Out-Null
-
-  $config = $FLUENTD_CONFIG.replace('NODE_NAME', (hostname))
-  $config | Out-File -FilePath $fluentd_config_file -Encoding ASCII
-  Log-Output "Wrote fluentd logging config to $fluentd_config_file"
+  $fluentbit_parser_file = "$LOGGINGAGENT_ROOT\conf\parsers.conf"
+  $PARSERS_CONFIG | Out-File -FilePath $fluentbit_parser_file -Encoding ASCII
+  Log-Output "Wrote logging config to $fluentbit_parser_file"
 }
 
 # The NODE_NAME placeholder must be replaced with the node's name (hostname).
```

Comment:
Please remove this comment and the same one that's down below, and please double-check that there's nothing that we need to set dynamically in these two configs.